\RequirePackage[l2tabu]{nag}		% Warns for incorrect (obsolete) LaTeX usage

\documentclass[a4paper,12pt,openbib,oneside]{memoir} 
\usepackage{gensymb}
\usepackage{datetime}
\usepackage{ifpdf}
\usepackage{geometry}
\usepackage{booktabs}
\usepackage{adjustbox}
\usepackage{mathptmx} % Times New Roman-like font
\usepackage[T1]{fontenc}

% Set margins: 1.5in left, 1in top/right/bottom
\geometry{left=1.5in, right=1in, top=1in, bottom=1in}

\ifpdf
\pdfinfo{
   /Author (Mrs. Shaima Aslam Chaity)
   /Title (Undergrad Thesis)
   /Keywords (200121)
   /CreationDate (D:\pdfdate)
}
\fi
 
\ifdraftdoc 
	\usepackage{draftwatermark}				
	\SetWatermarkScale{0.3}
	\SetWatermarkText{\bf Draft: \today}
\fi

% Declare figure/table as a subfloat.
\frenchspacing
% Set 1.5 line spacing globally
\OnehalfSpacing 
% Sets numbering division level
\setsecnumdepth{subsection} 
\maxsecnumdepth{subsubsection}

% Chapter style (taken and slightly modified from Lars Madsen Memoir Chapter Styles)
\usepackage{calc,soul,fourier}
\makeatletter 
\newlength\dlf@normtxtw 
\setlength\dlf@normtxtw{\textwidth} 
\newsavebox{\feline@chapter} 
\newcommand\feline@chapter@marker[1][4cm]{%
	\sbox\feline@chapter{% 
		\resizebox{!}{#1}{\fboxsep=1pt%
			\colorbox{gray}{\color{white}\thechapter}% 
		}}%
		\rotatebox{90}{% 
			\resizebox{%
				\heightof{\usebox{\feline@chapter}}+\depthof{\usebox{\feline@chapter}}}% 
			{!}{\scshape\so\@chapapp}}\quad%
		\raisebox{\depthof{\usebox{\feline@chapter}}}{\usebox{\feline@chapter}}%
} 
\newcommand\feline@chm[1][4cm]{%
	\sbox\feline@chapter{\feline@chapter@marker[#1]}% 
	\makebox[0pt][c]{% aka \rlap
		\makebox[1cm][r]{\usebox\feline@chapter}%
	}}
\makechapterstyle{daleifmodif}{
	\renewcommand\chapnamefont{\normalfont\Large\scshape\raggedleft\so} 
	\renewcommand\chaptitlefont{\normalfont\Large\bfseries\scshape} 
	\renewcommand\afterchapternum{\par\vskip\midchapskip} 
}
\makeatother 
\chapterstyle{daleifmodif}

% Page style: page numbers at bottom center, headers for sections
\makepagestyle{myvf} 
\makeoddfoot{myvf}{}{\thepage}{} 
\makeevenfoot{myvf}{}{\thepage}{} 
\makeheadrule{myvf}{\textwidth}{\normalrulethickness} 
\makeevenhead{myvf}{\small\textsc{\leftmark}}{}{} 
\makeoddhead{myvf}{}{}{\small\textsc{\rightmark}}
\pagestyle{myvf}

% Command to clear to next odd page
\newcommand{\clearemptydoublepage}{\newpage{\thispagestyle{empty}\cleardoublepage}}

% Creates indexes for Table of Contents, List of Figures, List of Tables, and Index
\makeindex

\usepackage{import}
\usepackage{lipsum}
\usepackage{amsfonts}
\usepackage[centertags]{amsmath}
\usepackage{stmaryrd}
\usepackage{amssymb}
\usepackage{enumitem}
\usepackage{amsthm}
\usepackage{newlfont}
\usepackage{graphicx}
\usepackage{longtable,rotating}
\usepackage[applemac]{inputenc}
\usepackage{colortbl}
\usepackage{wasysym}
\usepackage{mathrsfs}
\usepackage{subcaption}
\usepackage{float}
\usepackage{multirow}
\usepackage{verbatim}
\usepackage{upgreek}
\usepackage{latexsym}
\usepackage[square,numbers,sort&compress]{natbib}
\usepackage{url}
\usepackage[spanish,english]{babel}
\usepackage{color}
\usepackage[colorlinks=true,allcolors=black]{hyperref}
\usepackage{memhfixc}
\usepackage{footnote}
\usepackage{microtype}
\usepackage{rotfloat}
\usepackage{alltt}
\usepackage[version=0.96]{pgf}
\usepackage{tikz}
\usepackage{graphicx} % For figure environment
\usepackage{float}
\usepackage{seqsplit}
\usetikzlibrary{arrows,shapes,automata,backgrounds,petri,topaths, shapes.geometric, arrows.meta}

% Reduce widows and orphans
\widowpenalty=1000
\clubpenalty=1000

% Custom commands
\newcommand{\keywords}[1]{\par\noindent{\small{\bf Keywords:} #1}}
\newcommand{\parcial}[2]{\frac{\partial#1}{\partial#2}}
\newcommand{\vectorr}[1]{\mathbf{#1}}
\newcommand{\vecol}[2]{\left(\begin{array}{c} \displaystyle#1 \\ \displaystyle#2 \end{array}\right)}
\newcommand{\mados}[4]{\left(\begin{array}{cc} \displaystyle#1 &\displaystyle #2 \\ \displaystyle#3 & \displaystyle#4 \end{array}\right)}
\newcommand{\pgftextcircled}[1]{%
    \setbox0=\hbox{#1}%
    \dimen0\wd0%
    \divide\dimen0 by 2%
    \begin{tikzpicture}[baseline=(a.base)]%
        \useasboundingbox (-\the\dimen0,0pt) rectangle (\the\dimen0,1pt);
        \node[circle,draw,outer sep=0pt,inner sep=0.1ex] (a) {#1};
    \end{tikzpicture}
}
\newcommand{\range}[1]{\textnormal{range }#1}
\newcommand{\innerp}[2]{\left\langle#1,#2\right\rangle}
\newcommand{\prom}[1]{\left\langle#1\right\rangle}
\newcommand{\tra}[1]{\textnormal{tra} \: #1}
\newcommand{\sign}[1]{\textnormal{sign\,}#1}
\newcommand{\sech}[1]{\textnormal{sech} #1}
\newcommand{\diag}[1]{\textnormal{diag} #1}
\newcommand{\arcsech}[1]{\textnormal{arcsech} #1}
\newcommand{\arctanh}[1]{\textnormal{arctanh} #1}
\newcommand{\blackged}{\hfill$\blacksquare$}
\newcommand{\whiteged}{\hfill$\square$}
\newcounter{proofcount}
\renewenvironment{proof}[1][\proofname.]{\par
 \ifnum \theproofcount>0 \pushQED{\whiteged} \else \pushQED{\blackged} \fi%
 \refstepcounter{proofcount}
 \normalfont 
 \trivlist
 \item[\hskip\labelsep \itshape {\bf\em #1}]\ignorespaces
}{\addtocounter{proofcount}{-1} \popQED\endtrivlist}

% New square root definition
\let\oldsqrt\sqrt
\def\sqrt{\mathpalette\DHLhksqrt}
\def\DHLhksqrt#1#2{%
\setbox0=\hbox{$#1\oldsqrt{#2\,}$}\dimen0=\ht0
\advance\dimen0-0.2\ht0
\setbox2=\hbox{\vrule height\ht0 depth -\dimen0}%
{\box0\lower0.4pt\box2}}

% Caption styles
\newcommand{\mycaption}[2][\@empty]{%
	\captionnamefont{\scshape} 
	\changecaptionwidth
	\captionwidth{0.9\linewidth}
	\captiondelim{.\:} 
	\indentcaption{0.75cm}
	\captionstyle[\centering]{}
	\setlength{\belowcaptionskip}{10pt}
	\ifx \@empty#1 \caption{#2}\else \caption[#1]{#2}
}
\newcommand{\mysubcaption}[2][\@empty]{%
	\subcaptionsize{\small}
	\hangsubcaption
	\subcaptionlabelfont{\rmfamily}
	\sidecapstyle{\raggedright}
	\setlength{\belowcaptionskip}{10pt}
	\ifx \@empty#1 \subcaption{#2}\else \subcaption[#1]{#2}
}

% Initial letter
\usepackage{lettrine}
\newcommand{\initial}[1]{%
	\lettrine[lines=3,lhang=0.33,nindent=0em]{\color{gray}{\textsc{#1}}}{}}

% Theorem styles
\theoremstyle{plain}
\newtheorem{theo}{Theorem}[chapter]
\theoremstyle{plain}
\newtheorem{prop}{Proposition}[chapter]
\theoremstyle{plain}
\theoremstyle{definition}
\newtheorem{dfn}{Definition}[chapter]
\theoremstyle{plain}
\newtheorem{lema}{Lemma}[chapter]
\theoremstyle{plain}
\newtheorem{cor}{Corollary}[chapter]
\theoremstyle{plain}
\newtheorem{resu}{Result}[chapter]

% Hyphenation
\hyphenation{res-pec-tively}
\hyphenation{mono-ti-ca-lly}
\hyphenation{hypo-the-sis}
\hyphenation{para-me-ters}
\hyphenation{sol-va-bi-li-ty}

% new things added:

% Set no paragraph indentation and increase word spacing with justification
\setlength{\parindent}{0pt} % No indentation for paragraphs
\fontdimen2\font=0.3em % Nominal inter-word space
\fontdimen3\font=0.15em % Stretch for inter-word space
\fontdimen4\font=0.1em % Shrink for inter-word space
% \setlength{\spaceskip}{0.2em}
\setbeforesecskip{4.5ex plus 1ex minus 0.2ex} % Increases space before \section
\setbeforesubsecskip{4.25ex plus 1ex minus 0.2ex} % Increases space before \subsection

\setlength{\parskip}{12pt}

\begin{document}

\frontmatter
\pagenumbering{roman}
\input{frontmatter/title}
\input{frontmatter/Declaration.tex}
\input{frontmatter/certification}
\input{frontmatter/acknowledgment}
\input{frontmatter/abstract}

\renewcommand{\contentsname}{Table of Contents}
\maxtocdepth{subsection}
\tableofcontents*
\addtocontents{toc}{\par\nobreak \mbox{}\hfill{\bf Page}\par\nobreak}
\clearpage

\input{frontmatter/nomenclature.tex}
\clearpage

\listoftables
\addtocontents{lot}{\par\nobreak\textbf{{\scshape Table} \hfill Page}\par\nobreak}
\clearpage

\listoffigures
\addtocontents{lof}{\par\nobreak\textbf{{\scshape Figure} \hfill Page}\par\nobreak}
\clearpage

\mainmatter
\import{chapters/chapter01/}{chap01.tex}
\import{chapters/chapter02/}{chap02.tex}
\import{chapters/chapter03/}{chap03.tex}
\import{chapters/chapter04/}{chap04.tex}
\import{chapters/chapter05/}{chap05.tex}

\renewcommand\bibname{\fontsize{16pt}{1pt} \selectfont \centerline{REFERENCES} \global\def\bibname{REFERENCES}}

\begin{thebibliography}{99}

\bibitem{b1} Malik, S.E., Kanwal, S., Javed, J., Hidayat, W., Ghaffar, T., Aamir, A.H. (2023). Endocrine disorders in Beta-Thalassemia major patients at a Tertiary Care Hospital. \textit{Pakistan Journal of Medical Sciences}, 39(3), 726.

\bibitem{b2} Akiki, N., Hodroj, M.H., Bou-Fakhredin, R., Matli, K., Taher, A.T. (2023). Cardiovascular complications in $\beta$-thalassemia: getting to the heart of it. \textit{Thalassemia Reports}, 13(1), 38-50.

\bibitem{b3} Podder, N.K., Rana, H.K., Azam, M.S., Rana, M.S., Akhtar, M.R., Rahman, M.R., Rahman, M.H., Moni, M.A. (2020). A system biological approach to investigate the genetic profiling and comorbidities of type 2 diabetes. \textit{Gene Reports}, 21, 100830.

\bibitem{b4} Cao, A., \& Galanello, R. (2010). Beta-thalassemia. \textit{Genetics in Medicine}, 12(2), 61-76.

\bibitem{b5} Harrison, N.A., Kopelman, M.D., David, A.S., Fleminger, S. (2009). Endocrine diseases and metabolic disorders. In: \textit{Lishman's Organic Psychiatry: A Textbook of Neuropsychiatry}. 4th ed., Chichester: Wiley-Blackwell, pp. 617-688.

\bibitem{b6} Wuche, C. (2022). The cardiovascular system and associated disorders. \textit{British Journal of Nursing}, 31(17), 886-892.

\bibitem{b7} Carsote, M., Vasiliu, C., Trandafir, A.I., Albu, S.E., Dumitrascu, M.C., Popa, A., Mehedintu, C., Petca, R.C., Petca, A., Sandru, F. (2022). New entityâ€”thalassemic endocrine disease: major beta-thalassemia and endocrine involvement. \textit{Diagnostics}, 12(8), 1921.

\bibitem{b8} Akiki, N., Hodroj, M.H., Bou-Fakhredin, R., Matli, K., Taher, A.T. (2023). Cardiovascular complications in $\beta$-thalassemia: getting to the heart of it. \textit{Thalassemia Reports}, 13(1), 38-50.

\bibitem{b9} Ali, S., Mumtaz, S., Shakir, H.A., Khan, M., Tahir, H.M., Mumtaz, S., Mughal, T.A., Hassan, A., Kazmi, S.A.R., Sadia, Irfan, M. (2021). Current status of beta-thalassemia and its treatment strategies. \textit{Molecular Genetics \& Genomic Medicine}, 9(12), e1788.

\bibitem{b10} Podder, N.K., Shill, P.C., Rana, H.K., Omit, S.B.S., Al Shahriar, M.M.H., Azam, M.S. (2021). Genetic effects of COVID-19 on the development of neurodegenerative diseases. In: \textit{2021 5th International Conference on Electrical Information and Communication Technology (EICT)}, pp. 1-6. IEEE.

\bibitem{b11} Nain, Z., Rana, H.K., Lio, P., Islam, S.M.S., Summers, M.A., Moni, M.A. (2021). Pathogenetic profiling of COVID-19 and SARS-like viruses. \textit{Briefings in Bioinformatics}, 22(2), 1175-1196.

\bibitem{b12} Khare, S., Chatterjee, T., Gupta, S., Patel, A. (2023). Molecular pathways and gene networks in fetal hemoglobin as a novel protein target for beta-thalassemia. \textit{MGM Journal of Medical Sciences}, 10(3), 485-492.

\bibitem{b13} Sarkar, M.S., Mia, M.M., Al Amin, M., Hossain, M.S., Islam, M.Z. (2023). Bioinformatics and network biology approach to identifying type 2 diabetes genes and pathways that influence the progression of breast cancer. \textit{Heliyon}, 9(5).

\bibitem{b14} Nain, Z., Barman, S.K., Sheam, M.M., Syed, S.B., Samad, A., Quinn, J.M., Karim, M.M., Himel, M.K., Roy, R.K., Moni, M.A., Biswas, S.K. (2021). Transcriptomic studies revealed pathophysiological impact of COVID-19 to predominant health conditions. \textit{Briefings in Bioinformatics}, 22(6), bbab197.

\bibitem{b15} Datta, R., Podder, N.K., Rana, H.K., Islam, M.K.B., Moni, M.A. (2020). Bioinformatics approach to analyze gene expression profile and comorbidities of gastric cancer. In: \textit{2020 23rd International Conference on Computer and Information Technology (ICCIT)}, pp. 1-6. IEEE.

\bibitem{b16} Deshpande, R., VanderSluis, B., Myers, C.L. (2013). Comparison of profile similarity measures for genetic interaction networks. \textit{PLOS One}, 8(7), e68664.

\bibitem{b17} Kuleshov, M.V., Jones, M.R., Rouillard, A.D., Fernandez, N.F., Duan, Q., Wang, Z., Koplev, S., Jenkins, S.L., Jagodnik, K.M., Lachmann, A., McDermott, M.G. (2016). Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. \textit{Nucleic Acids Research}, 44(W1), W90-W97.

\bibitem{b18} Podder, N.K., Shill, P.C. (2022). Statistical and Bioinformatics Model to Identify the Influential Genes and Comorbidities of Glioblastoma. In: \textit{International Conference on Machine Intelligence and Emerging Technologies}, pp. 763-774. Springer, Cham.

\bibitem{b19} Chen, E.Y., Tan, C.M., Kou, Y., Duan, Q., Wang, Z., Meirelles, G.V., Clark, N.R., Ma'ayan, A. (2013). Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. \textit{BMC Bioinformatics}, 14(1), 128.

\bibitem{b20} Demo Reference 20. Placeholder entry for future use.

\bibitem{b21} Demo Reference 21. Placeholder entry for future use.

\bibitem{b22} Rana, H.K., Akhtar, M.R., Islam, M.B., Ahmed, M.B., Lio, P., Quinn, J.M., Huq, F., Moni, M.A. (2019). Genetic effects of welding fumes on the development of respiratory system diseases. \textit{Computers in Biology and Medicine}, 108, 142-149.

\bibitem{b23} Morrison, D.A. (2005). Networks in phylogenetic analysis: new tools for population biology. \textit{International Journal for Parasitology}, 35(5), 567-582.

\bibitem{b24} Rana, M.S., Podder, N.K., Rana, H.K., Hasan, M.I., Azam, M.S., Rahim, M.A., Iqbal, S.H.S., Saha, S. (2023). Identification of genomic associations between Parkinson's and neurodegenerative diseases using bioinformatics models. In: \textit{2023 International Conference on Electrical, Computer and Communication Engineering (ECCE)}, pp. 1-6. IEEE.

\bibitem{b25} Slavov, S.N., Haddad, S.K., Silva-Pinto, A.C., Amarilla, A.A., Alfonso, H.L., Aquino, V.H., Covas, D.T. (2012). Molecular and phylogenetic analyses of human Parvovirus B19 isolated from Brazilian patients with sickle cell disease and $\beta$-thalassemia major and healthy blood donors. \textit{Journal of Medical Virology}, 84(10), 1652-1665.

\bibitem{b26} Demo Reference 26. Placeholder entry for missing source.

\bibitem{b27} Durrani, I.A., Bhatti, A., John, P. (2023). Integrated bioinformatics analyses identifying potential biomarkers for type 2 diabetes mellitus and breast cancer: In SIK1-ness and health. \textit{PLOS One}, 18(8), e0289839.

\bibitem{b28} Demo Reference 28. Placeholder entry for missing source.

\bibitem{b29} Demo Reference 29. Placeholder entry for missing source.

\bibitem{b30} Talihati, Z., Abudurousuli, K., Hailati, S., Han, M., Nuer, M., Khan, N., Zhou, W. (2025). Screening of Hub Genes and Therapeutic Drugs in Cervical Cancer Using Integrated Bioinformatics Analysis. \textit{Journal of Cancer}, 16(1), 92.

\bibitem{b31} Demo Reference 31. Placeholder entry for missing source.

\bibitem{b32} Demo Reference 32. Placeholder entry for missing source.

\bibitem{b33} Demo Reference 33. Placeholder entry for missing source.

\bibitem{b34} Demo Reference 34. Placeholder entry for missing source.

\bibitem{b35} Barua, J.D., Omit, S.B.S., Rana, H.K., Podder, N.K., Chowdhury, U.N., Rahman, M.H. (2022). Bioinformatics and system biological approaches for the identification of genetic risk factors in the progression of cardiovascular disease. \textit{Cardiovascular Therapeutics}, 2022(1), 9034996.

\bibitem{b36} Rahman, M.H., Peng, S., Hu, X., Chen, C., Uddin, S., Quinn, J.M., Moni, M.A. (2019). Bioinformatics methodologies to identify interactions between type 2 diabetes and neurological comorbidities. \textit{IEEE Access}, 7, 183948-183970.

\end{thebibliography}

\end{document}